### ISSUE TITLE
how to update pdfmake version?

### ISSUE THREAD

**Issue by jandetlefsen** [xErik/pdfmake-browserified#7]
Looks like pdfmake 0.1.25 is on NPM. Could you update this project to the latest version? The 0.1.18 does not seem to support configuration of borders in tables, which i need for a layout. Happy to submit a pull request if you can explain how to update it.

**Comment by jandetlefsen** [comment]
I've forked it and run the build with 0.1.25 but getting an error when using it in the project 

`ReferenceError: fs is not defined
at Object.fontkit.openSync (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:365:14599)
at Function.t.open (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:424:236)
at o.font (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:463:525)
at FontProvider.provideFont (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:506:734)
at eval (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:533:1717)
at Array.forEach (native)
at measure (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:533:1247)
at TextTools.buildInlines (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:533:2325)
at DocMeasure.measureLeaf (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:500:2836)
at eval (webpack:///../~/pdfmake-browserified/build/pdfmake-browserified.min.js?:500:1889)`

my fork is here
https://github.com/jandetlefsen/pdfmake-browserified

**Comment by dmoebius** [comment]
@jandetlefsen: I see that you fixed the error in your fork. Great! :)
@xErik: could you please pull the changes from Jan and push a new release?
Or Jan, could you create a release from your fork?

**Comment by dmoebius** [comment]
Please ignore my last comment. I just saw that Jan did not fix the problem. Sorry, my fault.

So, is there any solution to this problem?

**Comment by jandetlefsen** [comment]
i've used this project, solves my problem https://github.com/EcoLogicAG/pdfmake-browserify

**Comment by dmoebius** [comment]
Yes, I noticed it, and already forked it to use 0.1.25 instead of 0.1.24. :)

**Comment by yanickrochon** [comment]
Whoever has this fork should submit a PR.
